app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt

```diff
@@ -0,0 +1,32 @@
+package fr.free.nrw.commons.utils
+
+import android.content.Context
+import android.content.pm.PackageManager
+import fr.free.nrw.commons.BuildConfig
+import java.util.*
+
+object ConfigUtils {
+    @JvmStatic
+    val isBetaFlavour: Boolean
+        get() = BuildConfig.FLAVOR == "beta"
```

Comment:
this doesn't need to be a `get()` as it doesn't need to be evaluated every time someone accesses it